txpool/txpool_test.go

```diff
@@ -96,11 +102,36 @@ func TestAppendAndRemove(t *testing.T) {
 	// Appending the same transaction again, should not return any error
 	assert.NoError(t, td.pool.AppendTx(testTrx))
 
-	td.pool.RemoveTx(testTrx.ID())
+	td.pool.removeTx(testTrx.ID())
 	assert.False(t, td.pool.HasTx(testTrx.ID()), "Transaction should be removed")
 	assert.Nil(t, td.pool.PendingTx(testTrx.ID()))
 }
 
+func TestCalculatingConsumption(t *testing.T) {
```

Comment:
@b00f What case need for check consumption map? I need to more details for test cases.